pincer/objects/guild/guild.py

```diff
@@ -575,9 +575,11 @@ async def list_guild_members(self, limit: int = 1, after: int = 0):
             the highest user id in the previous page |default| :data:`0`
         """
 
-        return await self._http.get(
+        members = await self._http.get(
             f"guilds/{self.id}/members?limit={limit}&after={after}"
         )
+        
+        return (GuildMember.from_dict(memeber) for member in members)
```

Comment:
aren't we supposed to use `yield`?

Reply:
We are

btw there's a typo, `memeber`

Reply:
@Sigmanificient Technically it does not matter since they both achieve the same result. 

Using `timeit` the speeds are
```py
# 0.275 ± 0.015
def returned(array):
    return (int(i) for i in array)
    
# 0.185 ± 0.003
def yielded(array):
    for i in array:
        yield int(i)

# 0.1825 ± 0.003
def yield_from(array):
    yield from (int(i) for i in array)
```

Suggestions?

Reply:
Not sure how you tested this, but these are my results :thinking:

**Array** | `returned` | `yielded` | `yield_from`
--- | --- | --- | ---
`[]` | 257 ± 0.893 (ns) | 157 ± 0.125 (ns) | 345 ± 1.85 (ns)
`[1]` | 369 ± 2.03 (ns) | 283 ± 2.07 (ns) | 497 ± 1.15 (ns)
`[*range(1000)]` | 104 µs ± 526 ns | 105 µs ± 1.12 µs | 124 µs ± 192 ns